### DISCUSSION TITLE
vc-drawings组件初始的按钮能隐藏吗

### DISCUSSION THREAD

**Discussion by shaopGao** [zouyaoji/vue-cesium#522] [Q&A] [answered]
<img width="409" alt="292ce89cbd178b204b3f61523b659d5" src="https://github.com/zouyaoji/vue-cesium/assets/115469092/2fa4e02a-cdc1-4c45-8576-ac451bf2bb8d">
如题，想用自己定义的工具栏上的按钮来绘制线，组件的按钮不需要了，怎么样能将其隐藏起来

**Comment by zouyaoji** [comment, marked as answer]
第一个按钮不能隐藏，其他可以收拢，如果第一个要隐藏就用v-if了。

收拢的话  mainFabOpts.modelValue 设为 false
